docs/tutorials/basic_ua_commands.md

```diff
@@ -0,0 +1,257 @@
+attached to an UA subscription.
+```
+This machine is not attached to a UA subscription.
+See https://ubuntu.com/advantage
+```
+
+### Attach
+
+To access any of those service offerings, the user need to attach UA to a subscription. This is
+achieved by running the attach command. Before we run it, we need to get ourselver a token.
+Any user with a Ubuntu One account is entitled to a free personal token to use with UA.
+You can retrieve your UA token from the [advantage](https://ubuntu.com/advantage/) portal.
+You will log in with your SSO credentials, the same credentials you use for https://login.ubuntu.com.
+After getting your UA token, go to the LXD container and run:
+
+```console
+$ sudo ua attach YOUR_TOKEN
+```
+
+It is expected for you to see an output similar to this one:
+
+```
+Enabling default service esm-infra
+Updating package lists
+UA Infra: ESM enabled
+This machine is now attached to 'USER ACCOUNT'
+
+SERVICE       ENTITLED  STATUS    DESCRIPTION
+cis           yes       disabled  Center for Internet Security Audit Tools
+esm-infra     yes       enabled   UA Infra: Extended Security Maintenance (ESM)
+fips          yes       n/a       NIST-certified core packages
+fips-updates  yes       n/a       NIST-certified core packages with priority security updates
+livepatch     yes       n/a       Canonical Livepatch service
+
+NOTICES
+Operation in progress: ua attach
+
+Enable services with: ua enable <service>
+
+                Account: USER ACCOUNT
+           Subscription: USER SUBSCRIPTION
+            Valid until: 9999-12-31 00:00:00+00:00
+Technical support level: essential
+```
+
+From this output, we can see that the attach command enables all of the services specified by the user subscription.
+After the command ends, we display what is the state of the machine now.
+That status output is exactly what you will see if you run the status command again.
+We can confirm this by running:
+
+```console
+$ ua status
+```
+
+One question that may arise is that the output of status and unattached are different. When
+attached, status presents two new columns, **ENTITLED** and **STATUS**, while also dropping the
+**AVAILABLE** column. For more information of why the output is different, please refer to this
+[explanation](../explanations/status_columns.md)
+
+Finally, another useful bit at the end of both attach and status is the contract expiration date:
+
+```
+    Account: USER ACCOUNT
+Subscription: USER SUBSCRIPTION
+Valid until: 9999-12-31 00:00:00+00:00
+```
+
+The `Valid until` field describe when your contract will be expired, so you can be aware of when it
+needs to be renewed.
+
+
+### Refresh
+
+In the last section, we mentioned that your contract can expire. After we renew the contract
+how can we make our machine aware of that ? We can do this through this command:
+
+```console
+$ sudo ua refresh
+```
+
+This command will refresh the contract on your machine. This command is also really useful if you
```

Comment:
Could we add the fact that `ua refresh` also updates the config esp. since `ua refresh --help` says that 
```sh
`ua refresh` is the equivalent of `ua refresh config && ua refresh contract`.
```

Reply:
That's a good point. I have not added more context about refresh here because I have not discussed in the tutorial about the config file or even MOTD messages. I think we can add a note at the end saying that refresh perform more actions on the machine and link this into an explanation doc. What do you all think ? @CalvoM @renanrodrigo @orndorffgrant

Reply:
just a text line here would do imho, mentioning it will also update UA configuration. When we have a Config tutorial, we can add a link here pointing to it.

Reply:
Okay, I will put another doc explaining what refresh does in the system and then I will reflect it on this section